generators/php/codegen/src/ast/core/Writer.ts

```diff
@@ -23,27 +26,36 @@ export class Writer extends AbstractWriter {
     public rootNamespace: string;
     /* Custom generator config */
     public customConfig: BasePhpCustomConfigSchema;
-
     /* Import statements */
-    private references: Record<Namespace, ClassReference[]> = {};
+    public readonly references: Record<Namespace, ClassReference> = {};
```

Comment:
consider renaming `Namespace` type because these keys include the namespace + name, eg "fully qualified name"